src/chocolatey/infrastructure/licensing/LicenseValidation.cs

```diff
@@ -126,8 +126,6 @@ public static ChocolateyLicense validate()
                 chocolateyLicense.Name = license.Name;
                 chocolateyLicense.Id = license.UserId.to_string();
 

```

Comment:
```suggestion
```

Personal preference to remove the blank line as well, fine if we don't want to.

Reply:
I thought about that but seeing the rest of the code I thought it was more consistent to leave it. But looking at it again I think it makes sense. I'll remove it.